ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs

```diff
@@ -125,5 +145,63 @@ public void SetShadowRadius(TFrameworkElement view, double radius)
             effect.BlurRadius = radius;
             view.Effect = effect;
         }
+
+        private static void SetProjectionMatrix(TFrameworkElement view, JArray transforms)
+        {
+            var transformMatrix = TransformHelper.ProcessTransform(transforms);
+
+            var translateMatrix = Matrix3D.Identity;
+            var translateBackMatrix = Matrix3D.Identity;
+            if (!double.IsNaN(view.Width))
+            {
+                translateMatrix.OffsetX = -view.Width / 2;
+                translateBackMatrix.OffsetX = view.Width / 2;
+            }
+
+            if (!double.IsNaN(view.Height))
+            {
+                translateMatrix.OffsetY = -view.Height / 2;
+                translateBackMatrix.OffsetY = view.Height / 2;
+            }
+
+            var projectionMatrix = translateMatrix * transformMatrix * translateBackMatrix;
+            ApplyProjection(view, projectionMatrix);
+        }
+
+        private static void ApplyProjection(TFrameworkElement view, Matrix3D projectionMatrix)
+        {
+            if (!IsSimpleTranslationOnly(projectionMatrix))
+            {
+                throw new InvalidOperationException("ReactNative.Net46 does not support 3D transformations");
+            }
```

Comment:
Can we implement 3D transforms in a separate PR, if we can evoke a Yellow Box warning instea of the exception?